app/routes.py

```diff
@@ -9,21 +9,38 @@ def get_posts():
 
 
 @app.route('/addPost',methods=['POST'])

```

Comment:
There is a standard way of naming REST API routes  [Naming Convention](https://www.restapitutorial.com/lessons/restfulresourcenaming.html)

```suggestion
@app.route('/posts',methods=['POST'])

```